### PR TITLE
(feat) O3-3174: Move visit related tags into the Visit Header component.

### PR DESCRIPTION
**Pull request by Twiineenock**
## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-docs.openmrs.org/docs/frontend-modules/contributing.en-US#contributing-guidelines) label. See existing PR titles for inspiration.
- [x] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://om.rs/o3ui).
- [ ] My work includes tests or is validated by existing tests.

## Summary

With this PR, the visit note tag and other visit related tags are moved to the visit header component as suggested by [paulsonder](https://github.com/paulsonder) in the ticket description here [https://openmrs.atlassian.net/browse/O3-3174](https://openmrs.atlassian.net/browse/O3-3174)

## Screenshots

Before feat:
visit-tag-before

After feat:
visit-tags

## Related Issue

[https://openmrs.atlassian.net/browse/O3-3174](https://openmrs.atlassian.net/browse/O3-3174)

### CODE REVIEW

**Comment by denniskigen** [comment]
FYI @ciaranduffy

File: packages/esm-patient-chart-app/src/visit-header/visit-header.component.tsx [hunk 1]
```diff
@@ -98,6 +98,11 @@ const PatientInfo: React.FC<PatientInfoProps> = ({ patient }) => {
         <span className={styles.patientName}>{name} </span>
       )}
       <span className={styles.patientInfo}>{`${age(patient?.birthDate)}, ${getGender(patient?.gender)}`}</span>
+      <ExtensionSlot
+        name="visit-header-tags-slot"
+        state={{ patientUuid: patient?.id, patient }}
+        className={styles.tagsFlexRow}
+      />
```
Comment (brandones) [inline packages/esm-patient-chart-app/src/visit-header/visit-header.component.tsx:105]: Is that class necessary? It seems like the right styles are being brought by the visit-attribute-tags component.

I would rather this slot be called `visit-header-middle-slot`. Better to be less opinionated about what goes in it. 

Maybe eventually we could put the stuff below, from queue entry, into the slot as extensions—the code below feels a bit out of place. But that's a different PR.

**Review by brandones — CHANGES_REQUESTED** [review]

**Comment by brandones** [comment]
Ping @Twiineenock . Is it clear to you how to move forward with this?

**Comment by brandones** [comment]
Closing since @Twiineenock is unresponsive

**Pull request closed without merging**